Update Cloud Quickstart (#18950)
cockroachdb/docs: src/current/cockroachcloud/create-an-account.md

</section>

<section class="filter-content" markdown="1" data-scope="email">
1. On the [registration page](https://cockroachlabs.cloud/signup?referralId=docs_create_account), click **Sign up with email**.

> 1. Enter your **Work email** and a unique **Password** for CockroachDB Cloud.

Reviewer: Is there a reason we need to be this prescriptive?
```suggestion
1. Enter an email address and password for CockroachDB Cloud.
```

Author: Confused, because I thought I made the same edit! Thanks.

> 1. Select the checkbox to accept the [Terms of Service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [Privacy Policy](https://www.cockroachlabs.com/privacy).

Reviewer: ```suggestion
1. Accept the [Terms of Service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [Privacy Policy](https://www.cockroachlabs.com/privacy), then click **Get Started**.
```
If you take this suggestion, you can eliminate the next step.

<section class="filter-content" markdown="1" data-scope="email">
1. On the [registration page](https://cockroachlabs.cloud/signup?referralId=docs_create_account), click **Sign up with email**.
1. Enter an email address and password for CockroachDB Cloud.
1. Accept the [Terms of Service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [Privacy Policy](https://www.cockroachlabs.com/privacy), then click **Get Started**.

> 1. Click **Get Started**.

Reviewer: ```suggestion
```
